NEWS.md

```diff
@@ -1,3 +1,18 @@
+### 6.11.0 (2020-07-07):
+
+* Open-sourced agent under Apache 2.0 license
+  * Added CODE_OF_CONDUCT.md file
+  * Streamlined README.md file
+  * Updated CONTRIBUTING.md file
+  * Added additional guidance to bug report template
+  * Added copyright headers to all source files
+* Added Distributed Tracing option to config file used for first time customers
+* Converted some test files to Node-tap
+* Removed "hidden" and unused code injector diagnostic capability
+* Upgraded @grpc/grpc-js from 1.0.4 to 1.0.5
+* Upgraded escodegen from 1.14.1 to 1.14.2
+* Upgraded @newrelic/aws-sdk from 1.1.2 to 1.1.3
```

Comment:
This PR did not land (closed without merging).

```suggestion
```

Reply:
Fixed